examples/generic-st7xx-with-xpt2046.py

```diff
@@ -0,0 +1,99 @@
+import sys
+sys.path.append('.')
+from st77xx import *
```

Comment:
Today most examples are assumed to be platform agnostic and are automatically run by CI.  
Your examples [fails on the CI](https://github.com/lvgl/lv_binding_micropython/runs/5955418027?check_suite_focus=true), but I think it should be excluded because it cannot run on the unix port.

To exclude an example from tests simply add it to the [EXCLUDE_PATH on `run.sh`](https://github.com/lvgl/lv_binding_micropython/blob/28109f072a05c35de67b21edf20c7300a7ee9ceb/tests/run.sh#L10).